src/extension.ts

```diff
@@ -104,45 +117,77 @@ export function activate(ctx: sourcegraph.ExtensionContext): void {
 
                 // TODO: This CSV generation is not robust.
                 const results = data.search.results.results
-                const csvData = [
-                    [
-                        'Repository',
-                        'Repository external URL',
-                        'File path',
-                        'File URL',
-                        'File external URL',
-                        'Search matches',
-                    ],
-                    ...results.map(r => {
-                        const searchMatches = r.lineMatches
-                            .map(line =>
-                                line.offsetAndLengths
-                                    .map(offset =>
-                                        line.preview?.substring(
-                                            offset[0],
-                                            offset[0] + offset[1]
-                                        )
+                const resultType = '__typename'
```

Comment:
Looks like `resultType` is used in a single place, maybe would be good to inline it as it was before?

Reply:
I had it inline initially but it wouldn't let me pass the tslint test due to `object access via string literals is disallowed` :(